src/BlobField/index.js

```diff
@@ -113,7 +110,7 @@ export default function BlobField({ collapsed = false }) {
       style={collapsed ? {} : { width: parentWidth, height: parentHeight }}
     >
       <ArtistNav />
-      <canvas ref={animationEl} width={width} height={height} />
+      <canvas ref={animationEl} width={window.innerWidth} height={window.innerHeight} />
```

Comment:
This causes an issue when something else such as a hover or click modifies the state and triggers a rerender. 
However if we add back the `width` and `height` states  it works! I made a commit that reverts those two states. But I would love to hear why this is happening bc I seriously can't warp my head around it. 

I tried `<canvas ref={animationEl} width={width && console.log(width)} height={height} />` and I can see that the width is changing but the canvas respects the sizing assigned in `blob.js`. However if we switch to `<canvas ref={animationEl} width={window.innerWidth} height={window.innerHeight} />` that's not the case anymore. In both cases I can see that the value passed to `width={}` is being updated though!

If a55976a274857c9daf304348a613c6bc712d45ab isn't recommended or there's a better solution without writing a lot of code let me know.

![Screenshot_20200519_193742](https://user-images.githubusercontent.com/11205091/82399025-13746600-9a09-11ea-852c-21bf38335d9b.png)

Reply:
15741246e7d646e6e8fcede59eced3eceb8bcdb6 fixes by getting rid of size setting on html element
`<canvas ref={animationEl}/>`